otel/metadata.csv

```diff
@@ -53,4 +53,29 @@ otelcol_process_runtime_total_sys_memory_bytes,gauge,,byte,,Total bytes of memor
 otelcol_processor_batch_batch_size_trigger_send,count,,unit,,Number of times the batch was sent due to a size trigger,0,otel,batch size trigger send,
 otelcol_exporter_sent_log_records,count,,unit,,Number of log records successfully sent to destination,0,otel,exporter sent log records,
 otelcol_receiver_refused_log_records,count,,unit,,Number of log records that could not be pushed into the pipeline,0,otel,receiver refused log records,
-otelcol_receiver_accepted_log_records,count,,unit,,Number of log records successfully pushed into the pipeline,0,otel,receiver accepted log records,
\ No newline at end of file
+otelcol_receiver_accepted_log_records,count,,unit,,Number of log records successfully pushed into the pipeline,0,otel,receiver accepted log records,
+runtime.uptime,count,15,millisecond,,Milliseconds since application was initialized,0,otel,runtime uptime,
```

Comment:
Are these metrics supposed to have have an `otel` prefix? The validations is currently failing because of this.

Reply:
Update: added `process` and `runtime` as allowed prefixes.